cli.py

```diff
@@ -1,5 +1,31 @@
 import click
+from ml.base import MLConfig
+from ml.base.estimator import MLEstimator
 
-
+@click.group()
 def main():
     pass
+
+@click.command()
+@click.option('--config', help='The config file')
+def train(config):
+    settings = MLConfig.from_file(config)
+    # The command line action should override the one in config
+    settings.global_config.mode = 'train'
+    model = MLEstimator.from_config(settings)
+    model.train_fn()
+
+@click.command()
+@click.option('--config', help='The config file')
+def predict(config):
+    settings = MLConfig.from_file(config)
+    # The command line action should override the one in config
```

Comment:
Use the format of `# TODO(jxwulittlebean): The command line flags will override the one in config here.` Always have the last period. Same for the other comment place too.